core-feature-pack/src/main/resources/content/docs/contrib/scripts/init.d/wildfly-init-debian.sh

```diff
@@ -59,7 +59,7 @@ fi
 
 # Location of wildfly
 if [ -z "$JBOSS_HOME" ]; then
-	JBOSS_HOME="/opt/wildfly"
+	JBOSS_HOME="/usr/share/wildfly"
```

Comment:
This is not a native package, if /usr/share was approved in Fedora it should be changed in Fedora it self.
http://www.tldp.org/LDP/Linux-Filesystem-Hierarchy/html/opt.html

The scripts here are not just "examples", they are fully working startup scripts, I don't see the case to change the path just because was approved in Fedora, the scripts apply to both debian and red hat based distros.